src/main/java/codechicken/nei/ClientHandler.java

```diff
@@ -149,6 +149,7 @@ public static void preInit() {
         loadSerialHandlers();
         loadHeightHackHandlers();
         loadHiddenHandlers();
+        loadEnableAutoFocus();
```

Comment:
Oh bugger, I completely forgot that, thanks!  Should I PR it to master or just leave it for this PR?